hardened.pri

```diff
@@ -0,0 +1,22 @@
+load(configure)
+# Define common variables; these are used by config tests _and_ the actual build
+HARDENED_SANITIZE_FLAGS = -fsanitize=address
+# Also: -fsanitize=undefined -fsanitize=integer-divide-by-zero -fvtable-verify=std
```

Comment:
These are available in GCC 4.9, but not 4.8. Maybe they should be combined with SANITIZE_MORE? Can somebody research this?